sqlstore/src/main/resources/db/migration/V0030__project_shortName.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project ADD COLUMN shortName varchar(5) DEFAULT NULL;
```

Comment:
It might not be perfect, but having something there that may fail validation is better than null, which will always fail validation.

I guess the solution may also be "when you upgrade to this release, you must define short names for all of your projects"